<!DOCTYPE html>
<html lang="en">
  <!-- Head -->
  <head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    
    <!-- Metadata, OpenGraph and Schema.org -->




<!-- Standard metadata -->
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<title>
  
  
    
      publications | Konstantinos P. Chatzipapas
    
  
</title>
<meta name="author" content="Konstantinos P. Chatzipapas">
<meta name="description" content="publications by categories in reversed chronological order. generated by jekyll-scholar.">

  <meta name="keywords" content="Radiation Biology, Monte Carlo simulations, AI, Geant4, Geant4-DNA, GATE">










<!-- Bootstrap & MDB -->
<link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04">
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous">



<!-- Fonts & Icons -->
<link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5">
<link defer rel="stylesheet" href="/assets/css/scholar-icons.css?62b2ac103a88034e6882a5be5f3e2772">
<link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap">

<!-- Code Syntax Highlighting -->
<link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light">



<!-- Styles -->




  <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%F0%9F%A7%AC%E2%98%A2%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;">

<link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e">
<link rel="canonical" href="https://www.kons.gr/publications/">

<!-- Dark Mode -->
<script src="/assets/js/theme.js?9a0c749ec5240d9cda97bc72359a72c0"></script>

  <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark">
  <script>
    initTheme();
  </script>










  </head>

  <!-- Body -->
  <body class="fixed-top-nav ">
    <!-- Header -->
    <header>
  <!-- Nav Bar -->
  <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation">
    <div class="container">
      
        <a class="navbar-brand title font-weight-lighter" href="/">
          Konstantinos P. Chatzipapas
        </a>
      
      <!-- Navbar Toggle -->
      <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar top-bar"></span>
        <span class="icon-bar middle-bar"></span>
        <span class="icon-bar bottom-bar"></span>
      </button>

      <div class="collapse navbar-collapse text-right" id="navbarNav">
        <ul class="navbar-nav ml-auto flex-nowrap">
          

          <!-- About -->
          <li class="nav-item ">
            <a class="nav-link" href="/">about
              
            </a>
          </li>

          <!-- Other pages -->
          
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
              
                
                <li class="nav-item active">
                  <a class="nav-link" href="/publications/">publications
                    
                      <span class="sr-only">(current)</span>
                    
                  </a>
                </li>
              
            
          
            
          
            
              
                
                <li class="nav-item ">
                  <a class="nav-link" href="/repositories/">repositories
                    
                  </a>
                </li>
              
            
          
            
              
                
                <li class="nav-item ">
                  <a class="nav-link" href="/cv/">cv
                    
                  </a>
                </li>
              
            
          
            
          
            
          
            
          
          
            <!-- Search -->
            <li class="nav-item">
              <button id="search-toggle" title="Search" onclick="openSearchModal()">
                <span class="nav-link">ctrl k <i class="ti ti-search"></i></span>
              </button>
            </li>
          
          
            <!-- Toogle theme mode -->
            <li class="toggle-container">
              <button id="light-toggle" title="Change theme">
                <i class="ti ti-sun-moon" id="light-toggle-system"></i>
                <i class="ti ti-moon-filled" id="light-toggle-dark"></i>
                <i class="ti ti-sun-filled" id="light-toggle-light"></i>
              </button>
            </li>
          
        </ul>
      </div>
    </div>
  </nav>
  
    <!-- Scrolling Progress Bar -->
    <progress id="progress" value="0">
      <div class="progress-container">
        <span class="progress-bar"></span>
      </div>
    </progress>
  
</header>


    <!-- Content -->
    <div class="container mt-5" role="main">
      
        

<div class="post">
  <header class="post-header">
    <h1 class="post-title">publications</h1>
    <p class="post-description">publications by categories in reversed chronological order. generated by jekyll-scholar.</p>
  </header>

  <article>
    <!-- _pages/publications.md -->

<!-- Bibsearch Feature -->

<script src="/assets/js/bibsearch.js?1bc438ca9037884cc579601c09afd847" type="module"></script>

<p><input type="text" id="bibsearch" spellcheck="false" autocomplete="off" class="search bibsearch-form-input" placeholder="Type to filter"></p>

<div class="publications">

<h2 class="bibliography">2025</h2>
<ol class="bibliography">
<li>
<div class="row">
  
    <div class="col col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="ANDRADEMIRANDA2025111376" class="col-sm-8">
    <!-- Title -->
    <div class="title">GIRAFE: Glottal Imaging Dataset for Advanced Segmentation, Analysis, and Facilitative Playbacks Evaluation</div>
    <!-- Author -->
    <div class="author">
      

      
      Gustavo
            Andrade-Miranda, <em>Konstantinos
            Chatzipapas</em>, Julián D.
            Arias-Londoño, and
        <span class="more-authors" title="click to view 1 more author" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '1 more author' ? 'Juan I. Godino-Llorente' : '1 more author';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.html(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">1 more author</span>
      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Data in Brief</em>,  Feb 2025
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
        <a href="https://doi.org/https://doi.org/10.1016/j.dib.2025.111376" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a>
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      

      
      

      
      

      
      
      
    

    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>The advances in the development of Facilitative Playbacks extracted from High-Speed videoendo- scopic sequences of the vocal folds are hindered by a notable lack of publicly available datasets annotated with the semantic segmentations corresponding to the area of the glottal gap. This fact also limits the reproducibility and further exploration of existing research in this field. To address this gap, GIRAFE (Glottal Imaging Repository for Advanced Segmentation, Analysis, and Facilitative Playbacks Evaluation) is a data repository designed to facilitate the devel- opment of advanced techniques for the semantic segmentation, analysis, and fast evaluation of High-Speed videoendoscopic sequences of the vocal folds. The repository includes 65 high-speed videoendoscopic recordings from a cohort of 50 patients (30 female, 20 male). The dataset com- prises 15 recordings from healthy controls, 26 from patients with diagnosed voice disorders, and 24 with an unknown health condition. All of them were manually annotated by an expert, including the masks corresponding to the semantic segmentation of the glottal gap. The repository is also complemented with the automatic segmentation of the glottal area using different state-of-the-art approaches. This data set has already supported several studies, which demonstrates its usefulness for the development of new glottal gap segmentation algorithms from High-Speed-Videoendoscopic sequences to improve or create new Facilitative Playbacks. Despite these advances and others in the field, the broader challenge of performing an accurate and completely automatic semantic segmentation method of the glottal area remains open.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="ARCHER2025112448" class="col-sm-8">
    <!-- Title -->
    <div class="title">A multiscale nanodosimetric study of GCR protons and alpha particles in the organs of astronauts on the lunar surface</div>
    <!-- Author -->
    <div class="author">
      

      
      Jay W.
            Archer, Matthew J.
            Large, David
            Bolst, and
        <span class="more-authors" title="click to view 8 more authors" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '8 more authors' ? 'Dousatsu Sakata, Hoang Ngoc Tran, Konstantinos P. Chatzipapas, Vladimir Ivantchenko, Anatoly B. Rosenfeld, Sebastien Incerti, Jeremy M.C. Brown, Susanna Guatelli' : '8 more authors';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.html(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">8 more authors</span>
      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Radiation Physics and Chemistry</em>,  Apr 2025
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
        <a href="https://doi.org/https://doi.org/10.1016/j.radphyschem.2024.112448" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a>
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      

      
      

      
      

      
      
      
    

    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>The early DNA damage on the surface of the Moon due to GCR protons and alpha particles were assessed using a multiscale approach in Geant4. This consisted of three simulation stages. A periodic boundary conditions approach was used to obtain the radiation field on the surface and inside a proposed lunar habitat. The radiation field on the cellular scale was obtained in the organs of male and female astronauts using the ICRP145 tetrahedral mesh phantoms. This was subsequently simulated using a full human cell model in Geant4-DNA to obtain the early DNA damage. Geant4-DNA track structure ionisation models upper energy limits were extended to be able to model the interactions of the GCR at sub-cellular level, covering an energy range from a few eV up to 1TeV. Hadronic interactions and the modelling of induced radiochemical species were also implemented. The early DNA damage was assessed using the Geant4-DNA molecularDNA example. A greater yield of DNA damage was observed on the lunar surface compared with the habitat, and indirect damage due to induced hydroxyl radicals constituted most of the damage. This study demonstrates a complete simulation pipeline for the assessment of early DNA damage in astronauts in the space radiation environment.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="HEEMSKERK2025100908" class="col-sm-8">
    <!-- Title -->
    <div class="title">Position in proton Bragg curve influences DNA damage complexity and survival in head and neck cancer cells</div>
    <!-- Author -->
    <div class="author">
      

      
      Tim
            Heemskerk, Celebrity
            Groenendijk, Marta
            Rovituso, and
        <span class="more-authors" title="click to view 7 more authors" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '7 more authors' ? 'Ernst van der Wal, Wouter van Burik, Konstantinos Chatzipapas, Danny Lathouwers, Roland Kanaar, Jeremy M.C. Brown, Jeroen Essers' : '7 more authors';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.html(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">7 more authors</span>
      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Clinical and Translational Radiation Oncology</em>,  Mar 2025
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
        <a href="https://doi.org/https://doi.org/10.1016/j.ctro.2024.100908" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a>
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      

      
      

      
      

      
      
      
    

    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Background and purpose 
  Understanding the cellular and molecular effect of proton radiation, particularly the increased DNA damage complexity at the distal end of the Bragg curve, is current topic of investigation. This work aims to study in vitro clonogenic survival and DNA damage foci kinetics of a head and neck squamous cell carcinoma cell line at various positions along a double passively scattered Bragg curve. Complementary in silico studies are conducted to gain insights into the link between cell survival variations, experimentally yielded foci and the number and complexity of double strand breaks (DSBs). 
  Materials and methods
  Proton irradiations are performed at the HollandPTC R&amp;D proton beamline, using a double passively scattered setup. A custom water phantom setup is employed to accurately position the samples within the Bragg curve. FaDu cells are irradiated at the proximal 36 % point of the Bragg peak, (P36), proximal 80 % point of the Bragg peak (P80) and distal 20 % point of the Bragg peak (D20), with dose-averaged mean lineal energies (yD¯) of 1.10 keV/μm, 1.80 keV/μm and 7.25 keV/μm, respectively.
  Results
  Clonogenic survival correlates strongly with yD¯, showing similar survival for P36 (D37%=3.0 Gy) and P80 (D37%=2.9 Gy), but decreased survival for D20 (D37% = 1.6 Gy). D20 irradiated samples exhibit increased 53BP1 foci shortly after irradiation, slower resolution of the foci, and larger residual 53BP1 foci after 24 h, indicating unrepaired complex breaks. These experimental observations are supported by the in silico study which demonstrates that irradiation at D20 leads to a 1.7-fold increase in complex DSBs with respect to the total number of strand breaks compared to P36 and P80.
  Conclusions
  This combined approach provides valuable insights into the cellular and molecular effect of proton radiation, emphasizing the increased DNA damage complexity at the distal end of the Bragg curve, and has the potential to enhance the efficacy of proton therapy.</p>
      </div>
    

    

    
  </div>
</div>
</li>
</ol>
<h2 class="bibliography">2024</h2>
<ol class="bibliography">
<li>
<div class="row">
  
    <div class="col col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="CHATZIPAPAS2024104839" class="col-sm-8">
    <!-- Title -->
    <div class="title">Development of a novel computational technique to create DNA and cell geometrical models for Geant4-DNA</div>
    <!-- Author -->
    <div class="author">
      

      
      <em>Konstantinos P.
            Chatzipapas</em>, Hoang Ngoc
            Tran, Milos
            Dordevic, and
        <span class="more-authors" title="click to view 4 more authors" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '4 more authors' ? 'Dousatsu Sakata, Sebastien Incerti, Dimitris Visvikis, Julien Bert' : '4 more authors';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.html(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">4 more authors</span>
      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Physica Medica</em>,  Oct 2024
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
        <a href="https://doi.org/https://doi.org/10.1016/j.ejmp.2024.104839" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a>
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      

      
      

      
      

      
      
      
    

    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Background
  This study aimed to develop a novel human cell geometry for the Geant4-DNA simulation toolkit that explicitly incorporates all 23 chromosome pairs of the human cell. This approach contrasts with the existing, default human cell, geometrical model, which utilizes a continuous Hilbert curve.
  Methods
  A Python-based tool named “complexDNA” was developed to facilitate the design of both simple and complex DNA geometries. This tool was employed to construct a human cell geometry with individual pairs of chromosomes. Subsequently, the performance of this chromosomal model was compared to the standard human cell model provided in the “molecularDNA” Geant4-DNA example.
  Results
  Simulations using the new chromosomal model revealed minimal discrepancies in DNA damage yield and fragment size distribution compared to the default human cell model. Notably, the chromosomal model demonstrated significant computational efficiency, requiring approximately three times less simulation time to achieve equivalent results.
  Conclusions
  This work highlights the importance of incorporating chromosomal structure into human cell models for radiation biology research. The “complexDNA” tool offers a valuable resource for creating intricate DNA structures for future studies. Further refinements, such as implementing smaller voxels for euchromatin regions, are proposed to enhance the model’s accuracy.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="Fira02102024" class="col-sm-8">
    <!-- Title -->
    <div class="title">In vitro validation of helium ion irradiations as a function of linear energy transfer in radioresistant human malignant cells</div>
    <!-- Author -->
    <div class="author">
      

      
      Aleksandra M. Ristić
            Fira, Otilija D.
            Keta, Vladana D.
            Petković, and
        <span class="more-authors" title="click to view 11 more authors" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '11 more authors' ? 'Miloš Đorđević, Giada Petringa, Serena Fattori, Roberto Catalano, Giuseppe Pablo Cirrone, Giacomo Cuttone, Dousatsu Sakata, Ngoc Hoang Tran, Konstantinos Chatzipapas, Sebastien Incerti, Ivan M. Petrović' : '11 more authors';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.html(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">11 more authors</span>
      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    
    
    
    <div class="periodical">
      <em>International Journal of Radiation Biology</em>,  Jul 2024
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
        <a href="https://doi.org/10.1080/09553002.2024.2373752" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a>
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      

      
      

      
      

      
      
      
    

    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p> Based on considerable interest to enlarge the experimental database of radioresistant cells after their irradiation with helium ions, HTB140, MCF-7 and HTB177 human malignant cells are exposed to helium ion beams having different linear energy transfer (LET). The cells are irradiated along the widened 62 MeV/u helium ion Bragg peak, providing LET of 4.9, 9.8, 23.4 and 36.8 keV/µm. Numerical simulations with the Geant4 toolkit are used for the experimental design. Cell survival is evaluated and compared with reference γ-rays. DNA double strand breaks are assessed via γ-H2AX foci. With the increase of LET, surviving fractions at 2 Gy decrease, while RBE (2 Gy, γ) gradually increase. For HTB140 cells, above the dose of 4 Gy, a slight saturation of survival is observed while the increase of RBE (2 Gy, γ) remains unaffected. With the increase of LET the increase of γ-H2AX foci is revealed at 0.5 h after irradiation. There is no significant difference in the number of foci between the cell lines for the same LET. From 0.5 to 24 h, the number of foci drops reaching its residual level. For each time point, there are small differences in DNA DSB among the three cell lines. Analyses of data acquired for the three cell lines irradiated by helium ions, having different LET, reveal high elimination capacity and creation of a large number of DNA DSB with respect to γ-rays, and are between those reported for protons and carbon ions. </p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="Beaudier12082024" class="col-sm-8">
    <!-- Title -->
    <div class="title">Quantitative analysis of dose dependent DNA fragmentation in dry pBR322 plasmid using long read sequencing and Monte Carlo simulations</div>
    <!-- Author -->
    <div class="author">
      

      
      Pierre
            Beaudier, Sara A.
            Zein, <em>Konstantinos
            Chatzipapas</em>, and
        <span class="more-authors" title="click to view 9 more authors" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '9 more authors' ? 'Hoang Ngoc Tran, Guillaume Devès, Laurent Plawinski, Rémy Liénard, Denis Dupuy, Philippe Barberet, Sébastien Incerti, Franck Gobet, Hervé Seznec' : '9 more authors';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.html(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">9 more authors</span>
      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Scientific Reports</em>,  Aug 2024
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
        <a href="https://doi.org/10.1038/s41598-024-69406-3" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a>
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      

      
      

      
      

      
      
      
    

    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Exposure to ionizing radiation can induce genetic aberrations via unrepaired DNA strand breaks. To investigate quantitatively the dose–effect relationship at the molecular level, we irradiated dry pBR322 plasmid DNA with 3 MeV protons and assessed fragmentation yields at different radiation doses using long-read sequencing from Oxford Nanopore Technologies. This technology applied to a reference DNA model revealed dose-dependent fragmentation, as evidenced by read length distributions, showing no discernible radiation sensitivity in specific genetic sequences. In addition, we propose a method for directly measuring the single-strand break (SSB) yield. Furthermore, through a comparative study with a collection of previous works on dry DNA irradiation, we show that the irradiation protocol leads to biases in the definition of ionizing sources. We support this scenario by discussing the size distributions of nanopore sequencing reads in the light of Geant4 and Geant4-DNA simulation toolkit predictions. We show that integrating long-read sequencing technologies with advanced Monte Carlo simulations paves a promising path toward advancing our comprehension and prediction of radiation-induced DNA fragmentation.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="TUANANH2024103422" class="col-sm-8">
    <!-- Title -->
    <div class="title">“dsbandrepair” – An updated Geant4-DNA simulation tool for evaluating the radiation-induced DNA damage and its repair</div>
    <!-- Author -->
    <div class="author">
      

      
      Le
            Tuan Anh, Tran
            Ngoc Hoang, Yann
            Thibaut, and
        <span class="more-authors" title="click to view 5 more authors" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '5 more authors' ? 'Konstantinos Chatzipapas, Dousatsu Sakata, Sébastien Incerti, Carmen Villagrasa, Yann Perrot' : '5 more authors';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.html(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">5 more authors</span>
      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Physica Medica</em>,  Jul 2024
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
        <a href="https://doi.org/https://doi.org/10.1016/j.ejmp.2024.103422" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a>
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      

      
      

      
      

      
      
      
    

    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Purpose
  Interdisciplinary scientific communities have shown large interest to achieve a mechanistic description of radiation-induced biological damage, aiming to predict biological results produced by different radiation quality exposures. Monte Carlo track-structure simulations are suitable and reliable for the study of early DNA damage induction used as input for assessing DNA damage. This study presents the most recent improvements of a Geant4-DNA simulation tool named “dsbandrepair”.
  Methods
  “dsbandrepair” is a Monte Carlo simulation tool based on a previous code (FullSim) that estimates the induction of early DNA single-strand breaks (SSBs) and double-strand breaks (DSBs). It uses DNA geometries generated by the DNAFabric computational tool for simulating the induction of early single-strand breaks (SSBs) and double-strand breaks (DSBs). Moreover, the new tool includes some published radiobiological models for survival fraction and un-rejoined DSB. Its application for a human fibroblast cell and human umbilical vein endothelial cell containing both heterochromatin and euchromatin was conducted. In addition, this new version offers the possibility of using the new IRT-syn method for computing the chemical stage.
  Results
  The direct and indirect strand breaks, SSBs, DSBs, and damage complexity obtained in this work are equivalent to those obtained with the previously published simulation tool when using the same configuration in the physical and chemical stages. Simulation results on survival fraction and un-rejoined DSB are in reasonable agreement with experimental data.
  Conclusions
  “dsbandrepair” is a tool for simulating DNA damage and repair, benchmarked against experimental data. It has been released as an advanced example in Geant4.11.2.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="Petrou10082024" class="col-sm-8">
    <!-- Title -->
    <div class="title">Investigation of Machine and Deep Learning Techniques to Detect HPV Status.</div>
    <!-- Author -->
    <div class="author">
      

      
      Efstathia
            Petrou, <em>Konstantinos
            Chatzipapas</em>, Panagiotis
            Papadimitroulas, and
        <span class="more-authors" title="click to view 5 more authors" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '5 more authors' ? 'Gustavo Andrade-Miranda, Paraskevi F. Katsakiori, Nikolaos D. Papathanasiou, Dimitris Visvikis, George C. Kagadis' : '5 more authors';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.html(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">5 more authors</span>
      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Journal of personalized medicine</em>,  Jul 2024
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
        <a href="https://doi.org/10.3390/jpm14070737" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a>
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      

      
      

      
      

      
      
      
    

    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>BACKGROUND: This study investigated alternative, non-invasive methods for human papillomavirus (HPV) detection in head and neck cancers (HNCs). We compared two  approaches: analyzing computed tomography (CT) scans with a Deep Learning (DL)  model and using radiomic features extracted from CT images with machine learning  (ML) models. METHODS: Fifty patients with histologically confirmed HNC were  included. We first trained a modified ResNet-18 DL model on CT data to predict  HPV status. Next, radiomic features were extracted from manually segmented  regions of interest near the oropharynx and used to train four ML models  (K-Nearest Neighbors, logistic regression, decision tree, random forest) for the  same purpose. RESULTS: The CT-based model achieved the highest accuracy (90%) in  classifying HPV status. Among the ML models, K-Nearest Neighbors performed best  (80% accuracy). Weighted Ensemble methods combining the CT-based model with each  ML model resulted in moderate accuracy improvements (70-90%). CONCLUSIONS: Our  findings suggest that CT scans analyzed by DL models hold promise for  non-invasive HPV detection in HNC. Radiomic features, while less accurate in this  study, offer a complementary approach. Future research should explore larger  datasets and investigate the potential of combining DL and radiomic techniques.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="Chatzipapas28052024" class="col-sm-8">
    <!-- Title -->
    <div class="title">Introduction of Hybrid Additive Manufacturing for Producing Multi-Material Artificial Organs for Education and In Vitro Testing</div>
    <!-- Author -->
    <div class="author">
      

      
      <em>Konstantinos
            Chatzipapas</em>, Anastasia
            Nika, and Agathoklis A.
            Krimpenis
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Designs</em>,  May 2024
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
        <a href="https://doi.org/10.3390/designs8030051" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a>
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      

      
      

      
      

      
      
      
    

    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>The evolution of 3D printing has ushered in accessibility and cost-effectiveness, spanning various industries including biomedical engineering, education, and microfluidics. In biomedical engineering, it encompasses bioprinting tissues, producing prosthetics, porous metal orthopedic implants, and facilitating educational models. Hybrid Additive Manufacturing approaches and, more specifically, the integration of Fused Deposition Modeling (FDM) with bio-inkjet printing offers the advantages of improved accuracy, structural support, and controlled geometry, yet challenges persist in cell survival, interaction, and nutrient delivery within printed structures. The goal of this study was to develop and present a low-cost way to produce physical phantoms of human organs that could be used for research and training, bridging the gap between the use of highly detailed computational phantoms and real-life clinical applications. To this purpose, this study utilized anonymized clinical Computed Tomography (CT) data to create a liver physical model using the Creality Ender-3 printer. Polylactic Acid (PLA), Polyvinyl Alcohol (PVA), and light-bodied silicone (Polysiloxane) materials were employed for printing the liver including its veins and arteries. In brief, PLA was used to create a mold of a liver to be filled with biocompatible light-bodied silicone. Molds of the veins and arteries were printed using PVA and then inserted in the liver model to create empty channel. In addition, the PVA was then washed out by the final product using warm water. Despite minor imperfections due to the printer’s limitations, the final product imitates the computational model accurately enough. Precision adjustments in the design phase compensated for this variation. The proposed novel low-cost 3D printing methodology successfully produced an anatomically accurate liver physical model, presenting promising applications in medical education, research, and surgical planning. Notably, its implications extend to medical training, personalized medicine, and organ transplantation. The technology’s potential includes injection training for medical professionals, personalized anthropomorphic phantoms for radiation therapy, and the future prospect of creating functional living organs for organ transplantation, albeit requiring significant interdisciplinary collaboration and financial investment. This technique, while showcasing immense potential in biomedical applications, requires further advancements and interdisciplinary cooperation for its optimal utilization in revolutionizing medical science and benefiting patient healthcare.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="mellak2024fasttrackf18positronpaths" class="col-sm-8">
    <!-- Title -->
    <div class="title">Fast-Track of F-18 Positron paths simulations using GANs</div>
    <!-- Author -->
    <div class="author">
      

      
      Youness
            Mellak, <em>Konstantinos
            Chatzipapas</em>, Alexandre
            Bousse, and
        <span class="more-authors" title="click to view 3 more authors" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '3 more authors' ? 'Catherine Chez-Le Rest, Dimitris Visvikis, Julien Bert' : '3 more authors';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.html(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">3 more authors</span>
      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    
    
    
    <div class="periodical">
      Mar 2024
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
        <a href="https://doi.org/https://doi.org/10.48550/arXiv.2403.06307" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a>
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      

      
      

      
      

      
      
      
    

    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>In recent years, the use of Monte Carlo (MC) simulations in the domain of Medical Physics has become a state-of-the-art technology that consumes lots of computational resources for the accurate prediction of particle interactions. The use of generative adversarial network (GAN) has been recently proposed as an alternative to improve the efficiency and extending the applications of computational tools in both medical imaging and therapeutic applications. This study introduces a new approach to simulate positron paths originating from Fluorine 18 (18 F) isotopes through the utilization of GANs. The proposed methodology developed a pure conditional transformer least squares (LS)-GAN model, designed to generate positron paths, and to track their interaction within the surrounding material. Conditioning factors include the pre-determined number of interactions, and the initial momentum of the emitted positrons, as derived from the emission spectrum of 18 F. By leveraging these conditions, the model aims to quickly and accurately simulate electromagnetic interactions of positron paths. Results were compared to the outcome produced with Geant4 Application for Tomography Emission (GATE) MC simulations toolkit. Less than 10 % of difference was observed in the calculation of the mean and maximum length of the path and the 1-D point spread function (PSF) for three different materials (Water, Bone, Lung).</p>
      </div>
    

    

    
  </div>
</div>
</li>
</ol>
<h2 class="bibliography">2023</h2>
<ol class="bibliography">
<li>
<div class="row">
  
    <div class="col col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="Stamouli30082023" class="col-sm-8">
    <!-- Title -->
    <div class="title">Dosimetric Evaluation of 177Lu Peptide Receptor Radionuclide Therapy Using GATE and Planet Dose</div>
    <!-- Author -->
    <div class="author">
      

      
      Ioanna
            Stamouli, Thomas
            Nanos, <em>Konstantinos
            Chatzipapas</em>, and
        <span class="more-authors" title="click to view 6 more authors" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '6 more authors' ? 'Panagiotis Papadimitroulas, Lydia-Aggeliki Zoglopitou, Theodoros Kalathas, Paraskevi F. Katsakiori, Anna Makridou, George C. Kagadis' : '6 more authors';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.html(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">6 more authors</span>
      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Applied Sciences</em>,  Aug 2023
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
        <a href="https://doi.org/10.3390/app13179836" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a>
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      

      
      

      
      

      
      
      
    

    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>This study aimed to compare the commercial dosimetric software Planet® Dose (version 3.1.1) from DOSIsoft and the open-source toolkit GATE. Dosimetry was performed for six patients receiving 200 mCi of Lutathera® every 8 weeks for four treatment cycles. For the dose calculation with Planet®, SPECT/CT images were acquired at 4, 24, 72 and 192 h post-injection. After the registration of all the time points to T0, the organs of interest (OOIs) were segmented. Time-activity curves were produced and the absorbed dose was calculated using the bi- and tri-exponential fitting methods. Regarding GATE simulations, the SPECT images of the 24 h time point were utilized for the radiopharmaceutical biodistribution in the OOIs and the attenuation maps were produced using the CT images. For liver and spleen, the average relative difference between GATE and Planet® was 9.6% and 11.1% for biexponential and 12.4% and 30.5% for triexponential fitting, respectively. The right and left kidneys showed differences up to 10.7% and 10.4% for the biexponential and up to 60.6% and 11.9% for the triexponential model, respectively. The absorbed dose calculated with GATE, Planet®(bi-exp) and Planet®(tri-exp) was in agreement with the literature. The results of the bi-exponential fitting were similar to the GATE-resulted calculations, while the tri-exponential fitting had a higher relative difference.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="CHATZIPAPAS2023102613" class="col-sm-8">
    <!-- Title -->
    <div class="title">Geant4-DNA simulation of human cancer cells irradiation with helium ion beams</div>
    <!-- Author -->
    <div class="author">
      

      
      <em>Konstantinos
            Chatzipapas</em>, Milos
            Dordevic, Sara
            Zivkovic, and
        <span class="more-authors" title="click to view 12 more authors" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '12 more authors' ? 'Ngoc Hoang Tran, Nathanael Lampe, Dousatsu Sakata, Ivan Petrovic, Aleksandra Ristic-Fira, Wook-Geun Shin, Sara Zein, Jeremy M.C. Brown, Ioanna Kyriakou, Dimitris Emfietzoglou, Susanna Guatelli, Sebastien Incerti' : '12 more authors';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.html(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">12 more authors</span>
      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Physica Medica</em>,  Jun 2023
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
        <a href="https://doi.org/https://doi.org/10.1016/j.ejmp.2023.102613" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a>
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      

      
      

      
      

      
      
      
    

    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Purpose:
  This study aimed to develop a computational environment for the accurate simulation of human cancer cell irradiation using Geant4-DNA. New cell geometrical models were developed and irradiated by alpha particle beams to induce DNA damage. The proposed approach may help further investigation of the benefits of external alpha irradiation therapy.
  Methods:
  The Geant4-DNA Monte Carlo (MC) toolkit allows the simulation of cancer cell geometries that can be combined with accurate modelling of physical, physicochemical and chemical stages of liquid water irradiation, including radiolytic processes. Geant4-DNA is used to calculate direct and non-direct DNA damage yields, such as single and double strand breaks, produced by the deposition of energy or by the interaction of DNA with free radicals.
  Results:
  In this study, the “molecularDNA” example application of Geant4-DNA was used to quantify early DNA damage in human cancer cells upon irradiation with alpha particle beams, as a function of linear energy transfer (LET). The MC simulation results are compared to experimental data, as well as previously published simulation data. The simulation results agree well with the experimental data on DSB yields in the lower LET range, while the experimental data on DSB yields are lower than the results obtained with the “molecularDNA” example in the higher LET range.
  Conclusion:
  This study explored and demonstrated the possibilities of the Geant4-DNA toolkit together with the “molecularDNA” example to simulate the helium beam irradiation of cancer cell lines, to quantify the early DNA damage, or even the following DNA damage response.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="Eleftheriadis07042023" class="col-sm-8">
    <!-- Title -->
    <div class="title">A framework for prediction of personalized pediatric nuclear medical dosimetry based on machine learning and Monte Carlo techniques</div>
    <!-- Author -->
    <div class="author">
      

      
      Vasileios
            Eleftheriadis, Georgios
            Savvidis, Valentina
            Paneta, and
        <span class="more-authors" title="click to view 3 more authors" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '3 more authors' ? 'Konstantinos Chatzipapas, George C Kagadis, Panagiotis Papadimitroulas' : '3 more authors';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.html(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">3 more authors</span>
      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Physics in Medicine &amp; Biology</em>,  Apr 2023
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
        <a href="https://doi.org/10.1088/1361-6560/acc4a5" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a>
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      

      
      

      
      

      
      
      
    

    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Objective: A methodology is introduced for the development of an internal dosimetry prediction toolkit for nuclear medical pediatric applications. The proposed study exploits Artificial Intelligence techniques using Monte Carlo simulations as ground truth for accurate prediction of absorbed doses per organ prior to the imaging acquisition considering only personalized anatomical characteristics of any new pediatric patient. Approach: GATE Monte Carlo simulations were performed using a population of computational pediatric models to calculate the specific absorbed dose rates (SADRs) in several organs. A simulated dosimetry database was developed for 28 pediatric phantoms (age range 2–17 years old, both genders) and 5 different radiopharmaceuticals. Machine Learning regression models were trained on the produced simulated dataset, with leave one out cross validation for the prediction model evaluation. Hyperparameter optimization and ensemble learning techniques for a variation of input features were applied for achieving the best predictive power, leading to the development of a SADR prediction toolkit for any new pediatric patient for the studied organs and radiopharmaceuticals. Main results. SADR values for 30 organs of interest were calculated via Monte Carlo simulations for 28 pediatric phantoms for the cases of five radiopharmaceuticals. The relative percentage uncertainty in the extracted dose values per organ was lower than 2.7%. An internal dosimetry prediction toolkit which can accurately predict SADRs in 30 organs for five different radiopharmaceuticals, with mean absolute percentage error on the level of 8% was developed, with specific focus on pediatric patients, by using Machine Learning regression algorithms, Single or Multiple organ training and Artificial Intelligence ensemble techniques. Significance: A large simulated dosimetry database was developed and utilized for the training of Machine Learning models. The developed predictive models provide very fast results (&lt;2 s) with an accuracy &gt;90% with respect to the ground truth of Monte Carlo, considering personalized anatomical characteristics and the biodistribution of each radiopharmaceutical. The proposed method is applicable to other medical dosimetry applications in different patients’ populations.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="Chatzipapas13022023" class="col-sm-8">
    <!-- Title -->
    <div class="title">Simulation of DNA damage using Geant4-DNA: an overview of the “molecularDNA” example application</div>
    <!-- Author -->
    <div class="author">
      

      
      <em>Konstantinos P.
            Chatzipapas</em>, Ngoc Hoang
            Tran, Milos
            Dordevic, and
        <span class="more-authors" title="click to view 12 more authors" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '12 more authors' ? 'Sara Zivkovic, Sara Zein, Wook-Geun Shin, Dousatsu Sakata, Nathanael Lampe, Jeremy M. C. Brown, Aleksandra Ristic-Fira, Ivan Petrovic, Ioanna Kyriakou, Dimitris Emfietzoglou, Susanna Guatelli, Sébastien Incerti' : '12 more authors';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.html(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">12 more authors</span>
      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Precision Radiation Oncology</em>,  Feb 2023
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
        <a href="https://doi.org/https://doi.org/10.1002/pro6.1186" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a>
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      

      
      

      
      

      
      
      
    

    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Abstract Purpose The scientific community shows great interest in the study of DNA damage induction, DNA damage repair, and the biological effects on cells and cellular systems after exposure to ionizing radiation. Several in silico methods have been proposed so far to study these mechanisms using Monte Carlo simulations. This study outlines a Geant4-DNA example application, named “molecularDNA”, publicly released in the 11.1 version of Geant4 (December 2022). Methods It was developed for novice Geant4 users and requires only a basic understanding of scripting languages to get started. The example includes two different DNA-scale geometries of biological targets, namely “cylinders” and “human cell”. This public version is based on a previous prototype and includes new features, such as: the adoption of a new approach for the modeling of the chemical stage, the use of the standard DNA damage format to describe radiation-induced DNA damage, and upgraded computational tools to estimate DNA damage response. Results Simulation data in terms of single-strand break and double-strand break yields were produced using each of the available geometries. The results were compared with the literature, to validate the example, producing less than 5% difference in all cases. Conclusion: “molecularDNA” is a prototype tool that can be applied in a wide variety of radiobiology studies, providing the scientific community with an open-access base for DNA damage quantification calculations. New DNA and cell geometries for the “molecularDNA” example will be included in future versions of Geant4-DNA.</p>
      </div>
    

    

    
  </div>
</div>
</li>
</ol>
<h2 class="bibliography">2021</h2>
<ol class="bibliography">
<li>
<div class="row">
  
    <div class="col col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="Sarrut14052021" class="col-sm-8">
    <!-- Title -->
    <div class="title">Advanced Monte Carlo simulations of emission tomography imaging systems with GATE</div>
    <!-- Author -->
    <div class="author">
      

      
      David
            Sarrut, Mateusz
            Bała, Manuel
            Bardiès, and
        <span class="more-authors" title="click to view 31 more authors" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '31 more authors' ? 'Julien Bert, Maxime Chauvin, Konstantinos Chatzipapas, Mathieu Dupont, Ane Etxebeste, Louise M Fanchon, Sébastien Jan, Gunjan Kayal, Assen S Kirov, Paweł Kowalski, Wojciech Krzemien, Joey Labour, Mirjam Lenz, George Loudos, Brahim Mehadji, Laurent Ménard, Christian Morel, Panagiotis Papadimitroulas, Magdalena Rafecas, Julien Salvadori, Daniel Seiter, Mariele Stockhoff, Etienne Testa, Carlotta Trigila, Uwe Pietrzyk, Stefaan Vandenberghe, Marc-Antoine Verdier, Dimitris Visvikis, Karl Ziemons, Milan Zvolský, Emilie Roncali' : '31 more authors';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.html(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">31 more authors</span>
      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Physics in Medicine &amp; Biology</em>,  May 2021
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
        <a href="https://doi.org/10.1088/1361-6560/abf276" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a>
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      

      
      

      
      

      
      
      
    

    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Built on top of the Geant4 toolkit, GATE is collaboratively developed for more than 15 years to design Monte Carlo simulations of nuclear-based imaging systems. It is, in particular, used by researchers and industrials to design, optimize, understand and create innovative emission tomography systems. In this paper, we reviewed the recent developments that have been proposed to simulate modern detectors and provide a comprehensive report on imaging systems that have been simulated and evaluated in GATE. Additionally, some methodological developments that are not specific for imaging but that can improve detector modeling and provide computation time gains, such as Variance Reduction Techniques and Artificial Intelligence integration, are described and discussed.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="Chatzipapa03032021" class="col-sm-8">
    <!-- Title -->
    <div class="title">IDDRRA: A novel platform, based on Geant4-DNA to quantify DNA damage by ionizing radiation</div>
    <!-- Author -->
    <div class="author">
      

      
      <em>Konstantinos P.
            Chatzipapas</em>, Panagiotis
            Papadimitroulas, George
            Loudos, and
        <span class="more-authors" title="click to view 2 more authors" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '2 more authors' ? 'Niko Papanikolaou, George C. Kagadis' : '2 more authors';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.html(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">2 more authors</span>
      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Medical Physics</em>,  Mar 2021
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
        <a href="https://doi.org/https://doi.org/10.1002/mp.14817" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a>
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      

      
      

      
      

      
      
      
    

    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Purpose This study proposes a novel computational platform that we refer to as IDDRRA (DNA Damage Response to Ionizing RAdiation), which uses Monte Carlo (MC) simulations to score radiation induced DNA damage. MC simulations provide results of high accuracy on the interaction of radiation with matter while scoring the energy deposition based on state-of-the-art physics and chemistry models and probabilistic methods. Methods The IDDRRA software is based on the Geant4-DNA toolkit together with new tools that were developed for the purpose of this study, including a new algorithm that was developed in Python for the design of the DNA molecules. New classes were developed in C++ to integrate the GUI and produce the simulation’s output in text format. An algorithm was also developed to analyze the simulation’s output in terms of energy deposition, Single Strand Breaks (SSB), Double Strand Breaks (DSB) and Cluster Damage Sites (CDS). Finally, a new tool was developed to implement probabilistic SSB and DSB repair models using MC techniques. Results This article provides the first benchmarks that the user of the IDDRRA tool can use to validate the functionality of the software as well as to provide a starting point to produce different types of DNA simulations. These benchmarks incorporate different kind of particles (e-, e+, protons, electron spectrum) and DNA molecules. Conclusion We have developed the IDDRRA tool and demonstrated its use to study various aspects of the modeling and simulation of a DNA irradiation experiment. The tool is expandable and can be expanded by other users with new benchmarks and applications based on the user’s needs and experience. New functionality will be added over time, including the quantification of the indirect damage.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="Chatzipapas22102021" class="col-sm-8">
    <!-- Title -->
    <div class="title">Standardization and Validation of Brachytherapy Seeds’ Modelling Using GATE and GGEMS Monte Carlo Toolkits</div>
    <!-- Author -->
    <div class="author">
      

      
      <em>Konstantinos P.
            Chatzipapas</em>, Dimitris
            Plachouris, Panagiotis
            Papadimitroulas, and
        <span class="more-authors" title="click to view 5 more authors" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '5 more authors' ? 'Konstantinos A. Mountris, Julien Bert, Dimitris Visvikis, Dimitris Mihailidis, George C. Kagadis' : '5 more authors';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.html(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">5 more authors</span>
      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Cancers</em>,  Oct 2021
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
        <a href="https://doi.org/10.3390/cancers13215315" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a>
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      

      
      

      
      

      
      
      
    

    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>This study aims to validate GATE and GGEMS simulation toolkits for brachytherapy applications and to provide accurate models for six commercial brachytherapy seeds, which will be freely available for research purposes. The AAPM TG-43 guidelines were used for the validation of two Low Dose Rate (LDR), three High Dose Rate (HDR), and one Pulsed Dose Rate (PDR) brachytherapy seeds. Each seed was represented as a 3D model and then simulated in GATE to produce one single Phase-Space (PHSP) per seed. To test the validity of the simulations’ outcome, referenced data (provided by the TG-43) was compared with GATE results. Next, validation of the GGEMS toolkit was achieved by comparing its outcome with the GATE MC simulations, incorporating clinical data. The simulation outcomes on the radial dose function (RDF), anisotropy function (AF), and dose rate constant (DRC) for the six commercial seeds were compared with TG-43 values. The statistical uncertainty was limited to 1% for RDF, to 6% (maximum) for AF, and to 2.7% (maximum) for the DRC. GGEMS provided a good agreement with GATE when compared in different situations: (a) Homogeneous water sphere, (b) heterogeneous CT phantom, and (c) a realistic clinical case. In addition, GGEMS has the advantage of very fast simulations. For the clinical case, where TG-186 guidelines were considered, GATE required 1 h for the simulation while GGEMS needed 162 s to reach the same statistical uncertainty. This study produced accurate models and simulations of their emitted spectrum of commonly used commercial brachytherapy seeds which are freely available to the scientific community. Furthermore, GGEMS was validated as an MC GPU based tool for brachytherapy. More research is deemed necessary for the expansion of brachytherapy seed modeling.</p>
      </div>
    

    

    
  </div>
</div>
</li>
</ol>
<h2 class="bibliography">2020</h2>
<ol class="bibliography">
<li>
<div class="row">
  
    <div class="col col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="KagadisEu2020" class="col-sm-8">
    <!-- Title -->
    <div class="title">Dosimetry and dosimetric challenges in paediatric radiology and radiotherapy.</div>
    <!-- Author -->
    <div class="author">
      

      
      George C.
            Kagadis, <em>Konstantinos
            Chatzipapas</em>, Theodora
            Kostou, and
        <span class="more-authors" title="click to view 2 more authors" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '2 more authors' ? 'Dimitris Plachouris, Panagiotis Papadimitroulas' : '2 more authors';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.html(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">2 more authors</span>
      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    
    
    
    <div class="periodical">
      <em>In ‘Radiosensitivity’ of children – Health issues after radiation exposure at young age – EU Scientific Seminar 2020</em>,  Dec 2020
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
        <a href="https://doi.org/10.2833/769468" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a>
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      

      
      

      
      

      
      
      
    

    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Patient specific dosimetry is of paramount importance in paediatric applications due to their significantly increased radiation sensitivity in comparison to adults. Paediatric patients have a higher risk to develop cancer compared to adults that obtain an equivalent amount of radiation dose. Exposure of children to ionising radiation has a longer expectancy in which it may provoke the development of radiation-induced issues such as cancer or as future parents, may increase hazard for passing on radiation-induced genetic disorders in the following generations. Considering the individual characteristics of children, it is important to reconsider diagnostic applications and therapy schemes in terms of personalisation and calculation of exact amount of absorbed dose per organ and patient. The TRS 457  (IAEA, 2007) recommendation, suggests that dose measurements should be made whenever possible, both on patients and phantoms. The use of phantoms enables repeatability and standardisation of measurements as well as a rapid evaluation of results, but not accurate and personalised dose estimation for a specific patient. Clinical measurements could complement the above outcome and are usually made using Thermoluminescent dosimeters (TLDs) or Optically Stimulated Luminescence dosimeters (OSLs). The main problem with TLDs is that there is typically a 0.1 mGy minimum absorbed dose to produce a reasonably accurate result. Modern techniques to assess organ and effective dose induced by medical imaging procedures are based on physical or computational phantoms.  Computer-based phantoms have the ability to represent a limitless amount of body types and have the potential to assess patient-specific organ and effective dose.  Recently, researchers have focused their studies on the production of an ideal ‘hybrid’ model as an evolution to computerised models. This hybrid model combines the realism of a voxelized phantom and the flexibility of a mathematical phantom. The anatomical virtual anthropomorphic models have been evolved a lot to successfully meet the standards of clinical practice.  Monte Carlo (MC) simulations on the other hand are broadly acknowledged as a fundamental technique to investigate the physics of nuclear medicine, radiology, radiation therapy, and dosimetry. When the MC methods are combined with precise paediatric computational phantoms, they may serve as the gold standard for the accurate assessment of absorbed dose in terms of personalised dosimetry. Organ absorbed dose is essential for paediatric patients, where major organ variations appear in comparison to adults’ anatomies.  The establishment and the optimisation of dosimetry protocols in paediatric acquisitions is therefore of high social and scientific interest. At the same time, there are critical concerns and multiple parameters to consider. The first challenge is the computational time and the need of extensive computing resources for the implementation of precise realistic MC simulations, with low statistical uncertainties, that consider patient specific characteristics. The second challenge is the establishment of several clinical protocols used in paediatric diagnosis and therapy based on individualised patient information. Finally, in clinical practice absolute radiation dose measurements to the various organs and to the patient in total is not an easy task to perform. The rapid evolution of computer science in terms of Artificial Intelligence can play a key role in the optimisation of the established paediatric clinical protocols in terms of personalisation.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="Chatzipapas26032020" class="col-sm-8">
    <!-- Title -->
    <div class="title">Ionizing Radiation and Complex DNA Damage: Quantifying the Radiobiological Damage Using Monte Carlo Simulations</div>
    <!-- Author -->
    <div class="author">
      

      
      <em>Konstantinos P.
            Chatzipapas</em>, Panagiotis
            Papadimitroulas, Dimitris
            Emfietzoglou, and
        <span class="more-authors" title="click to view 4 more authors" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '4 more authors' ? 'Spyridon A. Kalospyros, Megumi Hada, Alexandros G. Georgakilas, George C. Kagadis' : '4 more authors';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.html(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">4 more authors</span>
      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Cancers</em>,  Mar 2020
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
        <a href="https://doi.org/10.3390/cancers12040799" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a>
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      

      
      

      
      

      
      
      
    

    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Ionizing radiation is a common tool in medical procedures. Monte Carlo (MC) techniques are widely used when dosimetry is the matter of investigation. The scientific community has invested, over the last 20 years, a lot of effort into improving the knowledge of radiation biology. The present article aims to summarize the understanding of the field of DNA damage response (DDR) to ionizing radiation by providing an overview on MC simulation studies that try to explain several aspects of radiation biology. The need for accurate techniques for the quantification of DNA damage is crucial, as it becomes a clinical need to evaluate the outcome of various applications including both low- and high-energy radiation medical procedures. Understanding DNA repair processes would improve radiation therapy procedures. Monte Carlo simulations are a promising tool in radiobiology studies, as there are clear prospects for more advanced tools that could be used in multidisciplinary studies, in the fields of physics, medicine, biology and chemistry. Still, lot of effort is needed to evolve MC simulation tools and apply them in multiscale studies starting from small DNA segments and reaching a population of cells.</p>
      </div>
    

    

    
  </div>
</div>
</li>
</ol>
<h2 class="bibliography">2019</h2>
<ol class="bibliography">
<li>
<div class="row">
  
    <div class="col col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="Ricci01082019" class="col-sm-8">
    <!-- Title -->
    <div class="title">Monte Carlo Optical Simulations of a Small FoV Gamma Camera. Effect of Scintillator Thicknesses and Septa Materials</div>
    <!-- Author -->
    <div class="author">
      

      
      Rita
            Ricci, Theodora
            Kostou, <em>Konstantinos
            Chatzipapas</em>, and
        <span class="more-authors" title="click to view 6 more authors" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '6 more authors' ? 'Eleftherios Fysikopoulos, George Loudos, Luigi Montalto, Lorenzo Scalise, Daniele Rinaldi, Stratos David' : '6 more authors';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.html(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">6 more authors</span>
      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Crystals</em>,  Aug 2019
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
        <a href="https://doi.org/10.3390/cryst9080398" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a>
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      

      
      

      
      

      
      
      
    

    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Optical Monte Carlo simulations have been extensively used for the accurate modeling of light transport in scintillators for the improvement of detector designs. In the present work, a GATE Monte Carlo toolkit was used to study the effect of scintillator thicknesses and septa materials in the performance parameters evaluation of a commercially available small animal gamma-optical camera, named “γ-eye”. Firstly, the simulated γ-eye system was validated against experimental data. Then, part of the validated camera was modeled defining all of the optical properties by means of the UNIFIED model of GATE. Different CsI:Na scintillator crystals with varying thicknesses (from 4 mm up to 6 mm) and different reflector (septa) materials were simulated and compared in terms of sensitivity, light output and spatial resolution. Results have demonstrated the reliability of the model and indicate that the thicker crystal array presents higher sensitivity values, but degraded spatial resolution properties. Moreover, the use of black tape around crystals leads to an improvement in spatial resolution values compared to a standard white reflector material.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="Chatzipapas12112018" class="col-sm-8">
    <!-- Title -->
    <div class="title">Quantification of DNA double-strand breaks using Geant4-DNA</div>
    <!-- Author -->
    <div class="author">
      

      
      <em>Konstantinos P.
            Chatzipapas</em>, Panagiotis
            Papadimitroulas, Mohammad
            Obeidat, and
        <span class="more-authors" title="click to view 5 more authors" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '5 more authors' ? 'Kristen A. McConnell, Neil Kirby, George Loudos, Niko Papanikolaou, George C. Kagadis' : '5 more authors';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.html(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">5 more authors</span>
      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Medical Physics</em>,  Nov 2019
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
        <a href="https://doi.org/https://doi.org/10.1002/mp.13290" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a>
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      

      
      

      
      

      
      
      
    

    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Purpose This study aims to standardize the simulation procedure in measuring DNA double-strand breaks (DSBs), by using advanced Monte Carlo toolkits, and newly introduced experimental methods for DNA DSB measurement. Methods For the experimental quantification of DNA DSB, an innovative DNA dosimeter was used to produce experimental data. GATE in combination with Geant4-DNA toolkit were exploited to simulate the experimental environment. The PDB4DNA example of Geant4-DNA was upgraded and investigated. Parameters of the simulation such energy threshold (ET) for a strand break and base pair threshold (BPT) for a DSB were evaluated, depending on the dose. Results Simulations resulted to minimum differentiation in comparison to experimental data for ET = 19 ± 1 eV and BPT = 10 bp, and high differentiation for ET&lt;17.5 eV or ET&gt;22.5 eV and BPT = 10 bp. There was also small differentiation for ET = 17.5 eV and BPT = 6 bp. Uncertainty has been kept lower than 3%. Conclusions This study includes first results on the quantification of DNA double-strand breaks. The energy spectrum of a LINAC was simulated and used for the first time to irradiate DNA molecules. Simulation outcome was validated on experimental data that were produced by a prototype DNA dosimeter.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="Papadimitroulas17102018" class="col-sm-8">
    <!-- Title -->
    <div class="title">A Review on Personalized Pediatric Dosimetry Applications Using Advanced Computational Tools</div>
    <!-- Author -->
    <div class="author">
      

      
      Panagiotis
            Papadimitroulas, Theodora
            Kostou, <em>Konstantinos
            Chatzipapas</em>, and
        <span class="more-authors" title="click to view 12 more authors" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '12 more authors' ? 'Dimitris Visvikis, Konstantinos A. Mountris, Vincent Jaouen, Konstantinos Katsanos, Athanasios Diamantopoulos, Dimitris Apostolopoulos, Athanasios Balomenos, Yiannis Kopsinis, George Loudos, Christos Alexakos, Dimitris Karnabatidis, George C. Kagadis' : '12 more authors';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.html(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">12 more authors</span>
      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    
    
    
    <div class="periodical">
      <em>IEEE Transactions on Radiation and Plasma Medical Sciences</em>,  Oct 2019
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
        <a href="https://doi.org/10.1109/TRPMS.2018.2876562" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a>
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      

      
      

      
      

      
      
      
    

    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Pediatric patient-specific dosimetry of ionizing radiation is of great scientific and social interest. Children provide a higher relative cancer-risk from exposure to ionizing radiation compared to adults. The proposed study reviews the recent techniques applied in pediatric imaging and therapy applications for dosimetry purposes. Modern medicine makes use of advance computational tools for the personalization of internal and external dosimetry, especially in the sensitive group of children. Several groups of pediatric computational models have been developed which are combined with Monte Carlo (MC) simulations, machine learning (ML) techniques, and image processing algorithms for accurate dosimetry assessment. More specifically, this paper reviews the dosimetry applications in pediatric diagnostic procedures, including computed tomography and nuclear medicine applications. Right afterward, the most recent applications in therapeutic brachytherapy protocols are presented, which is a rather sensitive procedure in pediatrics. Finally, modern tools for dosimetry optimization are discussed, reviewing the most indicative applications with: 1) MC simulations for pediatric dosimetry assessment; 2) pediatric computational models, widely used in medical applications; and 3) ML techniques that provide an alternative method for estimating individualized absorbed doses.</p>
      </div>
    

    

    
  </div>
</div>
</li>
</ol>
<h2 class="bibliography">2012</h2>
<ol class="bibliography"><li>
<div class="row">
  
    <div class="col col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="kalogiannakis2012" class="col-sm-8">
    <!-- Title -->
    <div class="title">Playing on the journey of sound. A teaching proposal for children in early childhood</div>
    <!-- Author -->
    <div class="author">
      

      
      Michail
            Kalogiannakis, Charikleia
            Rekoumi, and <em>Konstantinos
            Chatzipapas</em>
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    
    
    
    <div class="periodical">
      <em>In Conference Proceedings 2012- Learning Science in the Society of Computers</em>,  Jun 2012
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      

      
      

      
      

      
      
      
    

    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>In this study, we develop an educational scenario for teaching sound to young children by using ICT (Information and Communication Technologies). The principles of the “New School” of the Greek Pedagogical Institute (recently called Institute of Educational Policy [IEP]) are mainly theoretical tools for creating this scenario. Sound is a very important thematic area of natural sciences for kindergarten. The proposed curricular intervention propels communication and interaction among children, and therefore the configuration of interpersonal relations allows emotional expression connected to music, as well as the exchange of thoughts. Also, it propels to children the possibility of exchanging arguments and negotiating their ideas and opinions. Specifically, in this scenario, through systematic activities for children, we developed certain ways of acquainting them with hearing, one of the most basic human senses. The main goal is to teach children to appreciate the value of hearing and to understand the fundamental methods of creating sound. Another intention is to teach children to become more sensitive with people with hearing problems, more sensitive to noise pollution and to be more respectful during quit hours. Sound, as teaching material, can extend the cultivation of incentives that activate the participation of young children. Furthermore, it can reclaim children’ previous knowledge and experiences and guide them to the configuration, design and organization of chances to enrich various learning fields. With the proposed scenario children produce original work using ICT as means of expression, like conceptual mapping software, websites, videos and other ICT tools.</p>
      </div>
    

    

    
  </div>
</div>
</li></ol>

</div>

  </article>

  

  
</div>

      
    </div>

    <!-- Footer -->
    
  <footer class="fixed-bottom" role="contentinfo">
    <div class="container mt-0">
      © Copyright 2025
      Konstantinos
      P.
      Chatzipapas. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>.  

      
      
        Last updated: February 21, 2025.
      
    </div>
  </footer>



    <!-- JavaScripts -->
    <!-- jQuery -->
<script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>

<!-- Bootsrap & MDB scripts -->
<script src="/assets/js/bootstrap.bundle.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script>


  <!-- Masonry & imagesLoaded -->
  <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script>
  <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script>
  <script defer src="/assets/js/masonry.js?a0db7e5d5c70cc3252b3138b0c91dcaf" type="text/javascript"></script>





















  <!-- Medium Zoom JS -->
  <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script>
  <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script>






<!-- Load Common JS -->
<script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script>
<script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script>
<script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script>

<!-- Jupyter Open External Links New Tab -->
<script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script>

<!-- Badges -->

  <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>


  <script async src="https://badge.dimensions.ai/badge.js"></script>



  <!-- MathJax -->
  <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script>
  
    <script src="/assets/js/mathjax-setup.js?a5bb4e6a542c546dd929b24b8b236dfd"></script>
    <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script>
  









  <!-- Scrolling Progress Bar -->
  <script defer src="/assets/js/progress-bar.js?2f30e0e6801ea8f5036fa66e1ab0a71a" type="text/javascript"></script>







  <!-- Back to Top -->
  <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script>
  <script>
    addBackToTop();
  </script>



  <!-- Search -->
  <script type="module" src="/assets/js/search/ninja-keys.min.js?a3446f084dcaecc5f75aa1757d087dcf"></script>
  <ninja-keys hidebreadcrumbs noautoloadmdicons placeholder="Type to start searching"></ninja-keys>
  <script src="/assets/js/search-setup.js?6c304f7b1992d4b60f7a07956e52f04a"></script>
  <script src="/assets/js/search-data.js"></script>
  <script src="/assets/js/shortcut-key.js?6f508d74becd347268a7f822bca7309d"></script>




  </body>
</html>
